### PR TITLE
Feature/other containers

### PR DESCRIPTION
**Pull request by neacsu**
This adds support to block explorer and network explorer containers as well, but with the caveat that you need the network explorer token for it to build

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -51,7 +51,40 @@ services:
                 depends_on:
                         - "genesis_validator"
 
+        mongo:
```
Comment (jstuczyn) [inline docker-compose.yml:54]: Wait, what's the reason for mongo here?

Comment (neacsu) [reply docker-compose.yml:54]: It's the mongo service from the block explorer docker setup. Maybe a more suitable naming is needed?

**Comment by samip5** [comment]
Helm chart for this too?

https://phoenixnap.com/kb/what-is-helm

tl;dr Related to Kubernetes, Helm charts are Helm packages consisting of YAML files and templates which convert into Kubernetes manifest files.

File: docker-compose.yml (hunk 1 above)
Comment (jstuczyn) [reply docker-compose.yml:54]: Oh, I didn't know block explorer was using mongo deep down. I just have personal bias against mongo : )

**Review by jstuczyn — APPROVED** [review]

**Comment by neacsu** [comment]
> Helm chart for this too?
> 
> https://phoenixnap.com/kb/what-is-helm
> 
> tl;dr Related to Kubernetes, Helm charts are Helm packages consisting of YAML files and templates which convert into Kubernetes manifest files.

Thank you for the suggestion. Kubernetes support is not currently on the roadmap, but if it will be at some point, we can consider Helm at that point.
